# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentParent.h
# Commit: bda570180adf
# Full Hash: bda570180adfcc20891d5cfb47406756fe8903ad
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2024-11-12 09:32:50
# Regressor Bug: 1928702
# File Overlap Count: 1
# Description:
#   Bug 1928702 - Part 11: Migrate ShowPersistentNotification to use IPC r=asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D227696
# ==============================================================================

diff -r c942992fa981 -r bda570180adf dom/ipc/ContentParent.h
--- a/dom/ipc/ContentParent.h	Mon Nov 11 20:34:15 2024 +0000
+++ b/dom/ipc/ContentParent.h	Mon Nov 11 20:34:15 2024 +0000
@@ -1011,9 +1011,6 @@
   mozilla::ipc::IPCResult RecvOpenNotificationSettings(
       nsIPrincipal* aPrincipal);
 
-  mozilla::ipc::IPCResult RecvNotificationEvent(
-      const nsAString& aType, const NotificationEventData& aData);
-
   mozilla::ipc::IPCResult RecvLoadURIExternal(
       nsIURI* uri, nsIPrincipal* triggeringPrincipal,
       nsIPrincipal* redirectPrincipal,